components/telemetry-controller/hack/boilerplate.go.txt

```diff
@@ -0,0 +1,15 @@
+/*
```

Comment:
Is there a special reason for this empty file?

Reply:
It's injected as a header to generated files

Reply:
Ah I see 👍🏻